lib/KTable/KTableGridItem.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+
+  <td
+    :class="$computedClass(coreOutlineFocus)"
+    :style="{ textAlign: textAlign, minWidth: minWidth, width: width }"
+    tabindex="0"
+    data-focus="true"
+    role="gridcell"
+    @keydown="onKeydown"
+  >
+    <slot :content="content">
+      {{ content }}
+    </slot>
+  </td>
+
+</template>
+
+
+<script>
+
+  import { DATA_TYPE_NUMERIC } from './useSorting';
+
+  export default {
+    name: 'KTableGridItem',
+    props: {
+      content: {
+        type: [String, Number, Boolean, Object, Array],
+        required: true,
+      },
+      dataType: {
+        type: String,
+        default: 'string',
+      },
+      rowIndex: {
+        type: Number,
+        required: true,
+      },
+      colIndex: {
+        type: Number,
+        required: true,
+      },
+      minWidth: {
+        type: String,
+        default: 'auto',
+      },
+      width: {
+        type: String,
+        default: 'auto',
+      },
+    },
+    computed: {
+      coreOutlineFocus() {
+        return {
+          ':focus': {
+            ...this.$coreOutline,
+            outlineOffset: '-2px',
+          },
+        };
+      },
+      textAlign() {
```

Comment:
As it's supposed to work same for both headers and data, now that we calculate this in `KTable`, could we just pass it down from `KTable` via prop to `KTableGridItem`?

Reply:
Then `textAlign` computed here can be removed

Reply:
done